test/manipulation.test.js

```diff
@@ -659,10 +659,60 @@ describe('manipulation', function() {
   });
 
   describe('updateOrCreate', function() {
+    var db = getSchema();
+    var Todo;
+
+    before(function setUp(done) {
+      Todo = db.define('Todo', {
+        content: String,
+      });
+      db.automigrate('Todo', done);
+    });
+    afterEach(function tearDown(done) {
+      db.automigrate(done);
```

Comment:
I think this is effectively a no-op, it definitely does not remove the `Todo` model (table, collection). I think it's safe to remove this whole afterEach hook.